2024-04-06 20:48:43,706 INFO    MainThread:28589 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2024-04-06 20:48:43,706 INFO    MainThread:28589 [wandb_setup.py:_flush():76] Configure stats pid to 28589
2024-04-06 20:48:43,706 INFO    MainThread:28589 [wandb_setup.py:_flush():76] Loading settings from /Users/artemon/.config/wandb/settings
2024-04-06 20:48:43,706 INFO    MainThread:28589 [wandb_setup.py:_flush():76] Loading settings from /Users/artemon/Library/Mobile Documents/com~apple~CloudDocs/Programming/python_projects/RNN/wandb/settings
2024-04-06 20:48:43,706 INFO    MainThread:28589 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-04-06 20:48:43,706 INFO    MainThread:28589 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-04-06 20:48:43,706 INFO    MainThread:28589 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-04-06 20:48:43,706 INFO    MainThread:28589 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-06 20:48:43,706 INFO    MainThread:28589 [wandb_init.py:_log_setup():521] Logging user logs to /Users/artemon/Library/Mobile Documents/com~apple~CloudDocs/Programming/python_projects/RNN/wandb/run-20240406_204843-jfgg5pxi/logs/debug.log
2024-04-06 20:48:43,706 INFO    MainThread:28589 [wandb_init.py:_log_setup():522] Logging internal logs to /Users/artemon/Library/Mobile Documents/com~apple~CloudDocs/Programming/python_projects/RNN/wandb/run-20240406_204843-jfgg5pxi/logs/debug-internal.log
2024-04-06 20:48:43,706 INFO    MainThread:28589 [wandb_init.py:_jupyter_setup():467] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x34e524bd0>
2024-04-06 20:48:43,706 INFO    MainThread:28589 [wandb_init.py:init():561] calling init triggers
2024-04-06 20:48:43,706 INFO    MainThread:28589 [wandb_init.py:init():568] wandb.init called with sweep_config: {}
config: {}
2024-04-06 20:48:43,706 INFO    MainThread:28589 [wandb_init.py:init():611] starting backend
2024-04-06 20:48:43,706 INFO    MainThread:28589 [wandb_init.py:init():615] setting up manager
2024-04-06 20:48:43,707 INFO    MainThread:28589 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-04-06 20:48:43,707 INFO    MainThread:28589 [wandb_init.py:init():623] backend started and connected
2024-04-06 20:48:43,709 INFO    MainThread:28589 [wandb_run.py:_label_probe_notebook():1299] probe notebook
2024-04-06 20:48:43,709 INFO    MainThread:28589 [wandb_run.py:_label_probe_notebook():1309] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-04-06 20:48:43,709 INFO    MainThread:28589 [wandb_init.py:init():715] updated telemetry
2024-04-06 20:48:43,709 INFO    MainThread:28589 [wandb_init.py:init():748] communicating run to backend with 90.0 second timeout
2024-04-06 20:49:17,280 WARNING MainThread:28589 [wandb_init.py:init():1203] interrupted
Traceback (most recent call last):
  File "/Users/artemon/.local/share/virtualenvs/RNN-j8Y16Los/lib/python3.11/site-packages/wandb/sdk/wandb_init.py", line 1181, in init
    run = wi.init()
          ^^^^^^^^^
  File "/Users/artemon/.local/share/virtualenvs/RNN-j8Y16Los/lib/python3.11/site-packages/wandb/sdk/wandb_init.py", line 751, in init
    result = run_init_handle.wait(
             ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/artemon/.local/share/virtualenvs/RNN-j8Y16Los/lib/python3.11/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/artemon/.local/share/virtualenvs/RNN-j8Y16Los/lib/python3.11/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/artemon/.local/share/virtualenvs/RNN-j8Y16Los/lib/python3.11/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.5/Frameworks/Python.framework/Versions/3.11/lib/python3.11/threading.py", line 622, in wait
    signaled = self._cond.wait(timeout)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.5/Frameworks/Python.framework/Versions/3.11/lib/python3.11/threading.py", line 324, in wait
    gotit = waiter.acquire(True, timeout)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
